scipy/sparse/linalg/tests/test_expm_multiply.py

```diff
@@ -76,20 +106,25 @@ def test_matrix_vector_multiply(self):
             observed = expm_multiply(A, v)
             expected = np.dot(scipy.linalg.expm(A), v)
             assert_allclose(observed, expected)
+            observed = estimated(expm_multiply)(aslinearoperator(A), v)
+            assert_allclose(observed, expected)
 
     def test_scaled_expm_multiply(self):
         np.random.seed(1234)
         n = 40
         k = 3
         nsamples = 10
-        for i in range(nsamples):
-            for t in (0.2, 1.0, 1.5):
-                with np.errstate(invalid='ignore'):
-                    A = scipy.linalg.inv(np.random.randn(n, n))
-                    B = np.random.randn(n, k)
-                    observed = _expm_multiply_simple(A, B, t=t)
-                    expected = np.dot(scipy.linalg.expm(t*A), B)
-                    assert_allclose(observed, expected)
+        for i, t in product(range(nsamples), [0.2, 1.0, 1.5]):
+            with np.errstate(invalid='ignore'):
+                A = scipy.linalg.inv(np.random.randn(n, n))
+                B = np.random.randn(n, k)
+                observed = _expm_multiply_simple(A, B, t=t)
+                expected = np.dot(scipy.linalg.expm(t*A), B)
```

Comment:
Use `expm`  from `sparse.linalg` and not expm as we are separating those two in this version.